up42/workflow.py

```diff
@@ -35,23 +35,26 @@ def __init__(self, auth: Auth, project_id: str, workflow_id: str):
         self.project_id = project_id
         self.workflow_id = workflow_id
         if self.auth.get_info:
-            self.info = self._get_info()
+            self._info = self.info
 
     def __repr__(self):
         return (
             f"Workflow(workflow_id={self.workflow_id}, project_id={self.project_id}, "
-            f"auth={self.auth}, info={self.info})"
+            f"auth={self.auth}, info={self._info})"
         )
 
-    def _get_info(self) -> Dict:
-        """Gets metadata info from an existing workflow"""
+    @property
+    def info(self) -> Dict:
+        """
+        Gets the workflow metadata information.
+        """
         url = (
             f"{self.auth._endpoint()}/projects/{self.project_id}/workflows/"
             f"{self.workflow_id}"
         )
         response_json = self.auth._request(request_type="GET", url=url)
-        self.info = response_json["data"]
-        return self.info
+        self._info = response_json["data"]
+        return self._info
```

Comment:
I think here we could just return `response_json["data"]`

Reply:
thanks